libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -976,11 +991,6 @@ - (SFAuthErrorHandlerList *)populateDefaultAuthErrorHandlerList
                                                    [weakSelf execFailureBlocks];
                                                    return YES;
                                                }
-                                               if (error.code == kSFOAuthErrorJWTInvalidGrant) {
```

Comment:
With the processing of the server-side error in `SFOAuthCoordinator`, this condition will be accurately ascertained through the `invalid_grant` conditional above this.